pallets/credit/src/lib.rs

```diff
@@ -1331,6 +1370,11 @@ pub mod pallet {
 
         /// update credit score by traffic
         fn update_credit_by_traffic(server_id: T::AccountId) {
+            let addrs = MaintainDevices::<T>::get();
```

Comment:
maybe not need add this judge.  
If device sending back and admin do not remove device account from list in time

Reply:
Good point, this check is not very important, will remove later.